scripts/cypress.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ${1:-run} = "fast" ]; then
+    ./scripts/cypress-fast-specs.sh && ./node_modules/.bin/cypress run
+elif [ ${1:-run} = "open" ]; then
```

Comment:
```suggestion
elif [ "$1" = "open" ]; then
```

same as above

Reply:
IIRC this is needed for cases where there is no argument. I.e. `yarn cy`.